### DISCUSSION TITLE
Transmogrifier Drop-down Set List not Working

### DISCUSSION THREAD

**Discussion by ghost** [SFX-WoW/Masque_Caith#32] [Issues] [answered]
When enable Masque: Caith plugin transmogrifier set list isn't expanding.

**Comment by StormFX** [comment]
Masque: Caith doesn't affect the transmogrifier UI.

**Comment by ghost** [comment]
I think this is an indirect bug.
A friend confirm same issue:
- Masque enabled no problem
- Masque Caith enabled transmogrifier set list isn't expanding.
For information "FR" client

I'll check for others addon side effect on this bug and tell you more

**Comment by StormFX** [comment, marked as answer]
I'll see if there's a typo in the frFR file.

Update: I just checked the locale files and with my client set to French. It works fine on my end. Disable everything but Masque and Masque: Caith.
